### PR TITLE
provider/google: Add alphaListed property to google accounts.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
@lwander please review.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java [hunk 1]
```diff
@@ -203,9 +218,13 @@ public String getApplicationName() {
...
     private final String projectName;
+    private final boolean alphaListed;
     private final String jsonKey;
     private final List<String> imageProjects;
     private final GoogleCredentials credentials;
     private final List<String> requiredGroupMembership;
     private final String applicationName;
+
+    private static final String COMPUTE_SERVICE_PATH = "compute/v1/projects/";
+    private static final String COMPUTE_ALPHA_SERVICE_PATH = "compute/alpha/projects/";
```
Comment (lwander) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java]: How do feel about making `v1`, `alpha`, etc an enum to switch between library versions? `v2` might be on the way, and perhaps there is a need for `beta`.

**Comment by lwander** [comment]
LGTM

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java (hunk 1 above)
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java]: I've dropped a todo and will revisit this.

**Pull request merged**